index.js

```diff
@@ -58,10 +58,18 @@ class TemplateCompiler extends Filter {
   }
 
   processString(string, relativePath) {
-    return 'export default ' + utils.template(this.options.templateCompiler, stripBom(string), {
-      contents: string,
-      moduleName: relativePath
-    }) + ';';
+    try {
+      return 'export default ' + utils.template(this.options.templateCompiler, stripBom(string), {
+        contents: string,
+        moduleName: relativePath
+      }) + ';';
+    } catch(error) {
+      // augment with location and type information and re-throw.
+      error.type = 'Template Compiler Error';
+      error.location = error.location.start;
```

Comment:
oh? `error` here comes from `ember-template-compiler`/`broccoli-persistent-filter` and I thought it always is going to be an object. As far as  `location` property goes, it should be there because https://github.com/glimmerjs/glimmer-vm/pull/540.

having said that, I'm happy to make it more resilient